src/clientsettingsdlg.cpp

```diff
@@ -360,14 +360,17 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     // custom directories
     QString strCustomDirectories = "<b>" + tr ( "Custom Directories" ) + ":</b> " +
                                    tr ( "If you need to add additional directories to the Connect dialog Directory drop down, "
-                                        "you can enter the addresses here.<br>"
-                                        "To remove a value, select it, delete the text in the input box, "
-                                        "then move focus out of the control." );
+                                        "you can enter the addresses here.<br>" );
 
     lblCustomDirectories->setWhatsThis ( strCustomDirectories );
     cbxCustomDirectories->setWhatsThis ( strCustomDirectories );
     cbxCustomDirectories->setAccessibleName ( tr ( "Custom Directories combo box" ) );
 
+    butDeleteCustomDirectories->setAccessibleName ( tr ( "Delete custom directory button" ) );
+    butDeleteCustomDirectories->setWhatsThis ( "<b>" + tr ( "Delete Custom Directory" ) + ":</b> " +
+                                               tr ( "Click the button to clear the currently selected custom directory "
```

Comment:
There's a specific usage difference between the Connect dialog and the Settings dialog.

On the Connect dialog, if no server is selected in the server list and the server combo box field is empty, hitting connect will default to the server hosting the current directory.

As you'll understand, that makes no sense for the Settings screen.  So the phrasing of the "What's This" needs to be a bit simpler: just refer to deleting the entry.